content/posts/2021/10/libplanet-0.11-0.19/index.eng.md

```diff
@@ -0,0 +1,251 @@
+---
+title: What's new in Libplanet 0.11–0.19
+date: 2021-10-29
+authors: [hong.minhee]
+---
+
+Hello.  Long time no see.  [Libplanet][][^1] has released several minor
+versions over the past few months.  However, as our release policy and cycle
+changed, and as we had responded to deal with diverse problems on the mainnet
+of our first game <cite>[Nine Chronicles]</cite>, which is powered by Libplanet, we'd been
```

Comment:
```suggestion
of our first game <cite>[Nine Chronicles]</cite>, powered by Libplanet, we'd been
```
The original sentence isn't grammatically incorrect. But I suggest shortening it this way for readability because the sentence is pretty long.